docs/how-to-use-the-playground.md

```diff
@@ -16,6 +16,17 @@ Depending on your network and computer, startup time may take 3-5 minutes. Once
 
 You first need to install git and docker-compose.
 
+## TCP ports used
+
+The playground runs a number of services. The TCP ports used may clash with existing services you run, such as MySQL or Postgres.
```

Comment:
`Postgres` ->  `PostgreSQL`?

Reply:
Postgres is a commonly used name for PostgreSQL

Reply:
We use PostgreSQL in other places. Should we unify them?

Reply:
Depending on the sentence it makes sense to use one or the other here "MySQL or PostgreSQL" is a little awkward/repetitive so I went with "MySQL or Postgres".

Reply:
OK.